docs/sources/administration/plugin-management/index.md

```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
 
 ### Install a plugin
 
-To install a plugin:
+To install a plugin you have multiple possibilities, the regular way is using Grafana UI.
 
 1. In Grafana, click **Administration > Plugins and data > Plugins** in the side navigation menu to view all plugins.
 1. Browse and find a plugin.
 1. Click the plugin's logo.
 1. Click **Install**.
 
-When the update is complete, you'll see a confirmation message that the installation was successful.
-
-### Update a plugin
+There are also additional ways to install plugins depending on your setup.
 
-To update a plugin:
-
-1. In Grafana, click **Administration > Plugins and data > Plugins** in the side navigation menu to view all plugins.
-1. Click the **Installed** filter to show only installed plugins.
-1. Click the plugin's logo.
-1. Click **Update**.
+#### Install a plugin using grafana cli
+Grafana CLI allows you to install, upgrade, and manage your Grafana plugins using a command line. For more information about Grafana CLI plugin commands, refer to [Plugin commands]({{< relref "../../cli/#plugins-commands" >}}).
 
-When the update is complete, you'll see a confirmation message that the update was successful.
 
-### Uninstall a plugin
+#### Install a plugin from a zip file
+This way is usually used for plugins not distributed via plugins catalog or in environments without internet connection.
```

Comment:
This method is typically used for plugins not available in the Plugin Catalog or in environments without internet access.